### ISSUE TITLE
A11Y: User Setting dialog keyboard interactions

### ISSUE THREAD

**Issue by UlianaMunich** [SAP/luigi#2453]
[DXP Issue-origin](https://github.tools.sap/dxp/backlog/issues/391)

Accessibility

The navigation menu list is not compliant yet with [Fiori Keyboard specifications](https://wiki.wdf.sap.corp/wiki/display/Acc/Fiori+Keyboard+Interaction?preview=/1842806926/1867396960/Fiori%20Keyboard%20Interaction%20(untagged).pdf) (p. 219-222)
- When the focus is on a list item, it should be able to use HOME to go to the first item and use END for the last item. In Fundamental NGX this is working fine.

The **select list** is not compliant with keyboard specifications. Fundamental NGX Select is working fine.
-  When the focus is set on the select (Language), it should be able to use ALT + DOWN to open the dropdown. Also use ALT + UP to close the dropdown when it's open.
-  After choosing a value from the select list, the focus should remain on the select.
-  When the select dropdown list is open, pressing ESC should only close the dropdown, not the entire dialog.
- 
UI:
- Add the possibility to show User Initials if available and no Account Profile picture is in pace
- Clicking on the Label "Language" should set the focus on the dropdown

UI references from UI5:https://sapui5.hana.ondemand.com/#/entity/sap.m.Select/sample/sap.m.sample.Select

_FOLLO UP TASK for select language dropdown:_
-  When opening the dropdown list, the focus should be already set on the selected list item.
-  When the dropdown is open, it should be able to navigate through the list with DOWN and UP. Not with TAB, because this should take focus to the "Save" button. (Follow up task #2565 )
